corehq/blobs/pillow.py

```diff
@@ -19,46 +14,28 @@
 
 class BlobDeletionProcessor(PillowProcessor):
 
-    def __init__(self, blob_db, bucket_base):
+    def __init__(self, blob_db):
         super(BlobDeletionProcessor, self).__init__()
         self.blob_db = blob_db
-        self.bucket_base = bucket_base
 
     def process_change(self, pillow_instance, change):
         if change.deleted:
-            bucket = join(self.bucket_base, change.id)
+            bucket = "{}/{}".format(change.meta.data_source_name, change.id)
```

Comment:
If I'm understanding this correctly [soft and hard deletions are always sent to kafka as soft deletions](https://github.com/dimagi/commcare-hq/blob/58e88da8458f1a271849bd02e2beb2b153ed70a2/corehq/apps/change_feed/pillow.py#L47). I'm not even sure what would need to be done to fix that. Would you propose to a `deleted` field to [`ChangeMeta`](https://github.com/dimagi/commcare-hq/blob/58e88da8458f1a271849bd02e2beb2b153ed70a2/corehq/ex-submodules/pillowtop/feed/interface.py#L7-L20) so kafka can distinguish between hard and soft deletions?